tests/jballerina-unit-test/src/test/resources/test-src/types/finaltypes/test_implicitly_final_negative.bal

```diff
@@ -44,3 +44,17 @@ listener test:MockListener ml = new (8080);
 public function testChangingListenerVariableAfterDefining() {
     ml = new test:MockListener(8081);
 }
+
+function testRestParamFinal(string p1, string... p2) {
+    p2 = ["a", "b"];
+}
+
+function (int a, int... b) testModuleLevelRestParamFinal = function (int i, int... b) {
+        b = [];
+    };
+
+public function testLocalLevelRestParamFinal() {
+    function (int a, int... b) func = function (int i, int... b) {
```

Comment:
One of these can use var.

Reply:
Do we have one with var? I meant `var f = ...`.

Reply:
Added in commit [#bf890dc](https://github.com/ballerina-platform/ballerina-lang/pull/41844/commits/bf890dc26035cc346247f1d2154c27869439a44f)